mindsdb/integrations/handlers/writer_handler/question_answer.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, args: dict, model_parameters: ModelParameters):
         retriever = get_retriever(
             embeddings_model_name=self.embeddings_model_name,
             persist_directory=self.persist_directory,
+            collection_name=args.get('collection_name', '_default_collection')
```

Comment:
provide the option for user to specify collection_name, defaults to default value if not provided.

This makes it a bit cleaner when wanting to work with multiple contexts in a given chromaDB instance